SWEgit/src/com/company/Main.java

```diff
@@ -198,4 +199,24 @@ public static void GetAverage(String[] arr) {
         System.out.println("Average : "+sum/arr.length);
         }
     }
+ public static void shuffle(String[] arr) {
```

Comment:
Please take copy from the array and change in it .... when we excute all the original array will be changed in your function so please take copy from the array and change it